ci-operator/jobs/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main-periodics.yaml

```diff
@@ -518,6 +518,53 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build04
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: main
+    org: shiftstack
+    repo: shiftstack-ci
+  interval: 5m
+  labels:
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-shiftstack-shiftstack-ci-main-monitor-nerc-dev
+  spec:
```

Comment:
Don't forget to add slack notification.

Reply:
We don't have that for monitoring jobs. They execute every minute, failures may become extremely noisy. Since this is used in Prometheus, I suggest settings alarms that side if we want that kind of alerts

Reply:
Oh my bad, I was under the impression our other monitoring jobs reported on slack. All good then.